src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -113,8 +122,36 @@ func New(location, namespace, resource string, kubeconfig string, forwardport st
 			"perconaservermongodbs",
 		)
 	}
+	sslSecrets := make([]sslSecret, 0)
 	filePaths := make([]string, 0)
-	if resourceType(resource) == "pxc" {
+	switch resource {
```

Comment:
it should be a`switch resourceType(resource)`.
with plain `switch resource`, it's not working (and breaks pxc) the new code breaks
`pt-k8s-debug-collector -cluster cluster1 -namespace pgo -resource pgv2`
the dumper receives pgv2/cluster1 string inside resource variable.